src/main/kotlin/org/rust/ide/inspections/import/AutoImportFix.kt

```diff
@@ -397,6 +397,7 @@ private val RsPath.namespaceFilter: (RsQualifiedNamedElement) -> Boolean get() =
     }
     is RsPathExpr -> { e ->
         when (e) {
+            is RsEnumItem -> isCompletion
```

Comment:
Not sure that it's only one place where we should do such change for completion but can't come up with other cases